content/_data/indexpage/supporters.yml

```
@@ -38,3 +38,15 @@
   url: https://www.tsinghua.edu.cn
 - name: XMission
   url: https://xmission.com
+- name: Belnet
+  url: https://belnet.be/
+- name: RWTH Aachen University
+  url: https://www.rwth-aachen.de/
+- name: Hostico
+  url: https://hostico.ro/
+- name: FreeDiff
```

I missed this in my earlier review
```suggestion
- name: FreeDif
```